packages/block-editor/src/components/block-list/style.scss

```
@@ -201,6 +201,7 @@
 			left: $border-width;
 			right: $border-width;
 			bottom: $border-width;
+			border-radius: $radius-block-ui - $border-width; // Border is outset, so so subtract the width to achieve correct radius.
```

There is a duplicate `so` in this and the other comments :)

```suggestion
			border-radius: $radius-block-ui - $border-width; // Border is outset, so subtract the width to achieve correct radius.
```

---

Ah, wondreful, thank you I will address that!